docs/content/basics.md

```diff
@@ -101,8 +125,8 @@ Now the application is deployed, you can make HTTP requests to it using the
 default route for the application:
 
 ```
-$ curl http://example.demo.localflynn.com
-Hello from Flynn on port 8080 from container db0440f7-19b4-4369-b79e-7a48dba415c2
+$ curl http://example.1.localflynn.com
+Hello from Flynn on port 8080 from container 87418feb-c863-4f6c-ab85-56f441b460c4
```

Comment:
Drop all these docs changes except for the addition of the `COMMAND` column in the output of `flynn ps` to keep the output consistent with deploying to a demo cluster.